testsuite/domain/src/test/resources/host-configs/host-secrets.xml

```diff
@@ -55,7 +55,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm"/>
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false"/>
```

Comment:
OK, I'll delete it after verifying a test run.

On Fri, Sep 11, 2015 at 12:47 PM, Brian Stansberry <notifications@github.com
> wrote:

> In testsuite/domain/src/test/resources/host-configs/host-secrets.xml
> <https://github.com/wildfly/wildfly-core/pull/1057#discussion_r39296851>:
>
> > @@ -55,7 +55,7 @@
> >
> >      <domain-controller>
> >          <!-- Remote domain controller configuration with a host and port -->
> > -        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm"/>
> > +        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false"/>
>
> I don't see any use of this file at all. From the root of the source tree:
>
> $ git grep secrets
> testsuite/shared/src/main/java/org/wildfly/test/security/VaultSession.java:
> + "keytool -genseckey -alias Vault -storetype jceks -keyalg AES -keysize
> 128 -storepass secretsecret -keypass secretsecret -keystore %s",
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wildfly/wildfly-core/pull/1057/files#r39296851>.
>